data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2286,6 +2286,16 @@ spec:
                       \n If this field is not set explicitly, it defaults to \"Classic\".
                       \ This default is subject to change over time."
                     type: string
+                  networkLoadBalancer:
+                    description: networkLoadBalancerParameters holds configuration
+                      parameters for an AWS network load balancer. Present only if
+                      type is NLB.
+                    properties:
+                      eip-allocations:
+                        items:
+                          type: string
+                        type: array
+                    type: object
```

Comment:
Cross referencing with my comment in the enhancement: https://github.com/openshift/enhancements/pull/1593#discussion_r1677263221

What about `networkLoadBalancer` is a child of `eipAllocations`?

Reply:
yes done